src/main/java/com/alesya/mafa/pet/MainInterface.java

```diff
@@ -15,46 +15,37 @@
 
     private IOperationInterface operationInterface = new OperationInterface();
     private Scanner scanner = new Scanner(System.in);
-    private String input;
+    private String input = "";
 
     @Override
     public void run() {
-
-        boolean flag;
         operationInterface.setScanner(scanner);
 
-        while (true) {
-
-            flag = false;
+        while (input.compareTo("exit") != 0) {
 
             System.out.println(INPUT_APP);
             input = scanner.nextLine().trim();
-            if (input.equals("exit")) {
-                break;
-            }
-            if (input.equals("list")) {
-                operationInterface.listPet();
-                flag = true;
-            }
 
-            if (input.equals("add")) {
-                add();
-                flag = true;
-            }
-            if (input.equals("update")) {
-                update();
-                flag = true;
-            }
-            if (input.equals("delete")) {
-                delete();
-                flag = true;
-            }
-
-            if (!flag) {
-                System.out.println(INCORRECT_VALUE);
+            switch (input) {
+                case "exit":
+                    break;
+                case "list":
+                    operationInterface.listPet();
+                    break;
+                case "add":
+                    add();
+                    break;
+                case "update":
+                    update();
+                    break;
+                case "delete":
+                    delete();
+                    break;
+                default:
+                    System.out.println(INCORRECT_VALUE);
+                    break;
```

Comment:
do we need it?

Reply:
yes, it is necessary to inform the user that he entered an invalid parameter.

Reply:
done